pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -419,7 +419,11 @@ self: super:
     };
   });
 
-  xf86inputkeyboard = brokenOnDarwin super.xf86inputkeyboard; # never worked: https://hydra.nixos.org/job/nixpkgs/trunk/xorg.xf86inputkeyboard.x86_64-darwin
+  xf86inputkeyboard = super.xf86inputkeyboard.overrideAttrs (attrs: {
+    meta = attrs.meta // {
+      platforms = lib.platforms.freebsd ++ lib.platforms.netbsd ++ lib.platforms.openbsd;
+    };
+  });
```

Comment:
Why did you drop it from `*-linux`?

Reply:
Because it explicitly says that linux is not a supported platform in the configure script.

Reply:
Ref: https://gitlab.freedesktop.org/xorg/driver/xf86-input-keyboard/-/blob/master/configure.ac#L61